modules/flowable-form-engine-configurator/src/test/java/org/flowable/form/engine/test/ProcessWithFormTest.java

```diff
@@ -125,37 +123,35 @@ public void startProcessWithFormWithoutValidationOnConfiguration(ProcessEngineCo
     public void throwExceptionValidationOnStartProcessWithoutVariables(RuntimeService runtimeService, RepositoryService repositoryService) {
         ProcessDefinition processDefinition = repositoryService.createProcessDefinitionQuery().processDefinitionKey("oneTaskWithFormSideEffectProcess")
             .singleResult();
-        assertThrows(RuntimeException.class,
-            () -> runtimeService.startProcessInstanceWithForm(processDefinition.getId(), "COMPLETE", null, "test"),
-            "validation failed"
-        );
+        assertThatThrownBy(() -> runtimeService.startProcessInstanceWithForm(processDefinition.getId(), "COMPLETE", null, "test"))
+                .isInstanceOf(RuntimeException.class)
+                .hasMessageContaining("validation failed");
```

Comment:
This worked entirely by accident :). The "validation failed" for `assertThrows` is the message to be shown for the failure (what `.as` is in AssertJ).